app/res/layout/gregorian_date_widget.xml

```diff
@@ -0,0 +1,117 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <RelativeLayout
+        android:layout_width="fill_parent"
+        android:layout_height="fill_parent"
+        android:layout_marginBottom="10dp"
+        android:layout_marginTop="10dp"
+        android:gravity="center">
+
+        <Button
+            android:id="@+id/dayupbtn"
+            android:layout_width="50dp"
+            android:layout_height="wrap_content"
+            android:background="@drawable/date_button_top"
+            android:text="+"
+            android:textSize="@dimen/text_largest"
+            android:textColor="@color/darkest_grey"/>
+
+        <EditText
+            android:id="@+id/daytxt"
+            android:layout_width="50dp"

```

Comment:
Make sure you take a look at these on small and x-large screens too - can help you find some if you need